src/Blocks/BlockBigFlower.h

```diff
@@ -97,13 +97,15 @@ class cBlockBigFlowerHandler:
 
 	virtual bool CanBeAt(cChunkInterface & a_ChunkInterface, const Vector3i a_RelPos, const cChunk & a_Chunk) override
 	{
-		if (a_RelPos.y <= 0)
+		if (!cChunkDef::IsValidHeight(a_RelPos.y - 1))
 		{
 			return false;
 		}
+
+		Vector3i BelowBlockPos = a_RelPos.addedY(-1);
```

Comment:
move this declaration before the call to IsValidHeight and use `BlockBelowPos.y` as the parameter to that call.